### PR TITLE
Create new home pages

### PR DESCRIPTION
**Pull request by ZzRanger**
## Overview

Created designs for new home pages. You can see the changes [here](https://nebula-planner-fm7pa2rmh-utdnebula.vercel.app/). Below are the mockups for the new home pages.  Feedback on the pages would be greatly appreciated. I also created 2 versions of the profile page.Please let me know in the comments which profile page is preferred.

Profile Page

Plans

### CODE REVIEW

**Comment by 1solomonwakhungu** [comment]
On smaller screens, the menu should be hidden or darken the background when the menu is open on smaller screens. Attached is what the a person sees on a smaller screen.

**Comment by 1solomonwakhungu** [comment]
Profile picture is too big on mobile screens.

**Review by syl15 — DISMISSED** [review]

**Comment by jasonappah** [comment]
+1 on Solomon's comments, a few other nits I wanted to mention. 

- The sidebar items and input field labels are still set in Roboto instead of Inter.
- The main header text, and sidebar text/icons aren't using the right color - they should be a dark blue instead of black (#1C2A6D).
- None of the sidebar items seem to have a selected state.

This is looking great, really happy to see this coming together.

File: components/newhome/Home.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  return (
+    <main className={classes.container}>
+      <section className="grid sm:grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-x-16 gap-y-10">
+        <div className="col-span-full text-black text-[40px] font-sans font-semibold">Plans</div>
+        <button
+          onClick={handleCreatePlan}
+          className="col-span-full text-white rounded-xl w-32 h-12 flex justify-center items-center flex-row bg-[#3E61ED]"
+        >
+          <div className="flex flex-row ">
+            <div className="text-3xl flex flex-col justify-center items-center">
```
Comment (jasonappah) [inline components/newhome/Home.tsx:61]:
```suggestion
            <div className="text-3xl flex flex-col justify-center items-center mr-2">
```
Adding a bit more margin on the icon. Not looking to have the button look exactly how it does in Figma, but I feel the icon needs a little more room to breathe.

File: components/newhome/Profile.tsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        <Avatar
+          alt="Remy Sharp"
+          sx={{ width: 120, height: 120, position: 'absolute', bottom: '-60px', left: '20px' }}
+        >
+          MK
+        </Avatar>
+      </section>
+      <section className="mt-40 flex flex-col gap-y-8 self-center items-center">
+        <article className="bg-white">
+          <div className="grid grid-cols-2 gap-x-32 gap-y-16 px-6 py-4">
```
Comment (innub) [inline components/newhome/Profile.tsx:23]: If we are considering mobile screens, this container should probably change to a flex column on smaller screens. We should probably design mobile screens on the Figma beforehand though.

**Comment by aanos-m** [comment]
First of all, very very happy with the progress. 
So on my end, I think we should add some padding between the "Delete" button and the message before it. I feel like it just looks like it was just mushed together. 

Capture

Secondly, on the credits page, I feel like we should add some spacing between the header and the Profile div/box and see how that looks and make both the "Profile" box/div and the "Delete My Account" div/box a little rounded on the edges like we have done for the other boxes on the Home and Profile page.

**Review by ZzRanger — COMMENTED** [review]
Quick notes about profile page: I disabled the "Email" field since updating email requires at the very least some re-authentication in order to authorize such an action. We don't have that logic implemented yet. I also disabled the "major" and "minor" fields since we don't use that information anywhere else in Planner.

**Review by syl15 — APPROVED** [review]

**Pull request merged**